src/transformers/modeling_utils.py

```diff
@@ -1839,6 +1842,7 @@ def from_pretrained(cls, pretrained_model_name_or_path: Optional[Union[str, os.P
         offload_state_dict = kwargs.pop("offload_state_dict", False)
         load_in_8bit = kwargs.pop("load_in_8bit", False)
         int8_threshold = kwargs.pop("int8_threshold", 6.0)
+        no_load_in_8bit_modules = kwargs.pop("no_load_in_8bit_modules", None)
```

Comment:
Would it make more sense to have this be a class variable of `PreTrainedModel` (like the `no_split` variable used for big model inference)? I'm afraid the user won't know what to set this too and it looks like it's something we should automatically handle?

Reply:
I don't have a strong opinion on that but this argument is optional because the function `get_keys_not_to_convert` should automatically take care of that except for some models like Jukebox where it is a bit trickier due to its architecture. 
In this case the user will just have to manually set which modules should be kept in their native precision and specify them in the `kwargs`, so I feel like it is a bit easier than having it as an argument of `PretrainedModel` because you would need to open a PR to add the feature.